proposals/lip-0050.md

```diff
@@ -171,14 +171,36 @@ registerBLSKeyParamsSchema = {
         "proofOfPossession": {
             "dataType": "bytes",
             "fieldNumber": 2
+        },
+        "generatorKey": {
```

Comment:
The `"required"` object in the schema has `"blsKey"` instead of `"generatorKey"`, needs to be changed.

Reply:
good catch! fixed.